docs/content/users/extend/custom-commands.md

```diff
@@ -17,14 +17,16 @@ The [`ddev mysql`](../usage/commands.md#mysql) runs the `mysql` client inside th
 
 ## Host Commands
 
-To provide host commands, place a Bash script in `.ddev/commands/host`. For example, a PhpStorm launcher to make the `ddev phpstorm` command might go in `.ddev/commands/host/phpstorm` with these contents:
+To provide host commands, place a Bash script in `.ddev/commands/host`. For example, a PhpStorm launcher to make the `ddev phpstorm` command might go in `.ddev/commands/host/phpstorm` with these contents. The OSTypes and HostBinaryExists annotations are optional, but are useful to prevent the command from showing up if it's not useful to the user.
```

Comment:
Those annotation keys seem like literal references, so you might want to wrap them in backticks:

```suggestion
To provide host commands, place a Bash script in `.ddev/commands/host`. For example, a PhpStorm launcher to make the `ddev phpstorm` command might go in `.ddev/commands/host/phpstorm` with these contents. The `OSTypes` and `HostBinaryExists` annotations are optional, but are useful to prevent the command from showing up if it's not useful to the user.
```